src/graph/test/SchemaTest.cpp

```diff
@@ -111,8 +111,36 @@ TEST_F(SchemaTest, metaCommunication) {
         auto code = client->execute(query, resp);
         ASSERT_EQ(cpp2::ErrorCode::SUCCEEDED, code);
     }
-
-    // Tag test
+    // Test create tag without prop
+    {
+        cpp2::ExecutionResponse resp;
+        std::string query = "CREATE TAG tag1()";
+        auto code = client->execute(query, resp);
+        ASSERT_EQ(cpp2::ErrorCode::SUCCEEDED, code);
+    }
+    {
+        cpp2::ExecutionResponse resp;
+        std::string query = "DESCRIBE TAG tag1";
+        auto code = client->execute(query, resp);
+        ASSERT_EQ(cpp2::ErrorCode::SUCCEEDED, code);
```

Comment:
I think we should check the detailed results of "Describe TAG tag1", i.e., the props is empty

Reply:
great